scripts/notify/index.js

```diff
@@ -82,7 +82,7 @@ function singularlize(text) {
 const packageMap = new Map();
 async function generatePackageMap() {
 	const packageRoot = new URL('../../packages/', import.meta.url);
-	const packages = await glob(['*/package.json', '*/*/package.json'], {
+	const packages = await glob('*/package.json', {
```

Comment:
Looking at the current structure from the repo, this glob should be enough, I don't think we have two level nested packages 🤔